fhir-bulkdata-webapp/pom.xml

```diff
@@ -200,7 +193,6 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
-            <scope>provided</scope>
```

Comment:
Would there ever be any terminology operations on this?

Reply:
maybe if we get a _typeFilter with some valueset modifiers like :in, :not-in.  potentially with an implicit valueset.

Reply:
That's the only case I can see it injecting a conflict.